cumulusci/robotframework/pageobjects/__init__.py

```diff
@@ -0,0 +1,17 @@
+from robot.libraries.BuiltIn import BuiltIn
+from .PageObjects import PageObjects  # noqa: F401
+from .baseobjects import BasePage, ListingPage, HomePage, DetailPage  # noqa: F401
+
+
+def pageobject(pageType, object_name=None):
```

Comment:
Can this be `page_type` for consistency with the keywords and with CumulusCI's overall code style?

Reply:
yes, good catch. Thanks.